pkg/ui/ccl/src/views/reports/containers/stores/encryption.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+
+import * as protos from "src/js/protos";
+
+interface EncryptionStatusProps {
+  store: protos.cockroach.server.serverpb.StoreDetails$Properties;
+}
+
+export default class EncryptionStatus extends React.Component<EncryptionStatusProps, {}> {
+
+  renderSimpleRow(header: string, value: string) {
+    return (
+      <tr className="stores-table__row">
+        <th className="stores-table__cell stores-table__cell--header">{header}</th>
+        <td className="stores-table__cell" title={value}><pre>{value}</pre></td>
+      </tr>
```

Comment:
should style this a bit to make it more readable. Add a file `encryption.styl` next to this tsx file, with contents like this: https://github.com/cockroachdb/cockroach/blob/07e2b77d390874e057ec99758c33167f410144c7/pkg/ui/src/views/cluster/containers/nodeOverview/nodeOverview.styl

Except`.stores-table` instead of just `.table`. Also make sure to import the `styl` file from this tsx file.

Reply:
Done, but with a fixed width for `th` as we end up with two different tables.
![encryption](https://user-images.githubusercontent.com/5901480/40514392-f7d6486a-5f76-11e8-91bb-1eb9f8451712.png)